src/cmds/testing/block_dev/block_dev_test.c

```diff
@@ -470,18 +489,31 @@ static int part_test(struct block_dev *bdev) {
 	return err;
 }
 
+int is_valid_argument(char *endptr, char *str) {
```

Comment:
`static int` is better because the function is local to this file.

Reply:
Let's fix this small thing an I think your contribution can be merged.

Reply:
Refactored the function a bit and updated the commit with suggested changes. :)